spring-cloud-generator/scripts/resources/googleapis_modification_string.txt

```diff
@@ -0,0 +1,14 @@
+_spring_cloud_generator_version = "4.1.5-SNAPSHOT" # {x-version-update:spring-cloud-gcp:current}
+
+maven_install(
+    artifacts = [
+        "com.google.api:gapic-generator-java:" + _gapic_generator_java_version,
```

Comment:
We probably don't need to install `gapic-generator-java` anymore, as `spring-cloud-generator` should already have it.

Reply:
Removed in [d9bb09f](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/d9bb09f1bc7eb9edd5e35ce0ed61028b866ac54e).